### PR TITLE
don't escape title - it is escaped by default in mustache

### PR DESCRIPTION
**Pull request by hakubo**
I have also improved whitespaces a bit

### CODE REVIEW

File: extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+			okCallback = function (expires, reason) {
+				$.post(wgScript + '?action=ajax&rs=ChatAjax&method=blockOrBanChat', {
 						userToBanId: userId,
 						time: expires,
 						reason: reason,
 						mode: 'global',
 						token: mw.user.tokens.get('editToken')
 					},
-					$.proxy(function(data) {
+					function () {
```
Comment (ghost) [inline extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js:13]: What is the benefit of not using the `proxy` function?

**Comment by ghost** [comment]
Just one question for the sake of curiosity. Otherwise, LG, tej. 👍

File: extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js (hunk 1 above)
Comment (hakubo) [inline extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js:13]: I would flip it :)
What is the benefit of USING the proxy function here?

Comment (hakubo) [inline extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js:13]: In addition I think if you use $.proxy with no second parameter
you bind 'this' to undefined

Comment (hakubo) [inline extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js:13]: nope it binds this to Window object that is available there anyway

**Comment by pawelchcki** [comment]
LGTM

**Comment by ludwikkazmierczak** [comment]
Tested and accepted

**Pull request merged**